packages/esm-commons-lib/src/api.resource.ts

```diff
@@ -177,7 +167,7 @@ export function fetchPatientComputedConcept_HIV_Status(patientUUID: string) {
 
 export function fetchPatientLastEncounter(patientUuid: string, encounterType) {
```

Comment:
@CynthiaKamau I still see fetch functions, I think most of them can be transformed to use `SWR` and make them hooks. The ones I think my be hard might be the ones that are using promise.all but we can get a work around on them.